### PR TITLE
Claude template: Comment and Record are two buttons, and the other one is always the way out

### PR DESCRIPTION
**Pull request by AkshilVT**
## What

Redesign of the annotate control in the claude template (replaces the split pill from #644):

- **Two separate buttons**: 💬 Comment (left) and 🎙 Record (right), each icon + word, own border. One mode at a time; the *other* button is always the exit:
  - Comment armed → mic seat becomes **✓ Done**: flushes pending notes to Claude (same prefill-and-submit the auto-send uses), then disarms.
  - Recording → Comment seat becomes **■ + live clock** in `--error` ink; clicking either button stops. Transcribing shows in the same seat (`.busy`).
  - Faces are all baked-in markup; the stylesheet swaps them off `#annbtn.on` / `#annrec.on` via `:has` — no extra writers.
- **Element/Point picker follows the mode**, not the recording: visible for typed comments too, slides in on arm, away on disarm. The selected tool survives a stop — visible state, not a hidden trap.
- **Composer card slimmed**: anchor chip (element/⌖ flip) and per-note mic removed from `#annpop` (both stylesheet copies + the whole `annMic*` block). Placeholder still names the target kind ("What about this spot?").
- **Collision-detected labels**: `annFitStrip` measures the row with words on — *plus the off-screen picker's reserved width, so arming never flips words→icons under the click* — and stamps `.tight` only on true overflow. ResizeObserver on the box + MutationObservers on the two content clusters (never on `#anntools` itself, whose class the verdict writes). No fixed breakpoint.

## Verified

- `pytest` — template suites green (`test_claude_annotation_modes`, `test_claude_shots`, `test_claude_kind`, `test_theme`); full sweep clean except pre-existing/env-only failures reproduced on clean main.
- Live browser (cmux) against the running worktree server: arm/Done flow, popup contents (no chip, no mic, kind-aware placeholder), Esc, point tool, geometry stability (buttons pixel-identical rest↔armed), word/icon state consistent across arming at 330/460/700px, zero console errors.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

---

> [!NOTE]
> **Medium Risk**
> Large, behavior-heavy changes to annotation UX (send/done ordering, picker visibility, composer dismissal) in a monolithic template; mitigated by extensive static wiring tests but still easy to miss edge cases in live flows.
> 
> **Overview**
> **Replaces the split annotate pill** with separate **Comment** and **Record** controls. One mode at a time; the other button is always the exit (Done / stop + clock), with faces driven by `#annbtn.on` / `#annrec.on` and `:has` CSS instead of extra DOM writers.
> 
> **Element/Point picker** now follows **armed comment mode** for typed notes and walkthroughs (not only while recording). Stopping a recording no longer hides the picker or resets the tool to element.
> 
> **Composer (`#annpop`)** drops the anchor chip (no in-card element/spot flip) and the per-note mic (`annMic*` removed). Target kind is only in the textarea placeholder; pin choice is up front via the strip picker (Alt still overrides).
> 
> **`annDone`** commits an open draft, flushes unsent notes via the existing prefill/auto-send path, then disarms—with **`annDoneBusy`** so double-Done cannot auto-send before a point crop finishes. Mousedown dismissal exempts `#anncta` / `#anntool` so Done does not close the composer first.
> 
> **`annFitStrip`** uses ResizeObserver + mutation observers (with off-screen picker width reserved) to add `.tight` and hide label words only on real overflow, not a fixed breakpoint.
> 
> Tests in `test_claude_annotation_modes`, `test_claude_shots`, and `test_claude_kind` are updated for the new wiring.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit bff936d1055443e97caf1e855936c6041114c4f7. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: fused_render/templates/claude/template.html [hunk 1]
```diff
@@ -6269,7 +6203,25 @@
...
+// open with words in it is committed FIRST — Done means "send what I said",
+// and the mousedown exemption above is what kept that draft alive this far —
+// through the same annCommit the Enter key uses (save, crop, auto-send).
+// Then any notes the auto-send guards left pending (a turn in flight, a
+// typed draft the prefill refused to touch) flush with the same
+// prefill-and-submit, and the mode goes away.
+async function annDone() {
+  if (annPop.style.display === "block" && annTa.value.trim()) {
+    await annCommit();
+  }
```
[the remark this answers is not in the document]
Comment (AkshilVT) [reply fused_render/templates/claude/template.html:6237]: Fixed in bff936d1: `annDone` now carries a busy guard — a second Done click during the commit's await is a no-op, so the note's save-time crop always lands before the auto-send.
